ra-tls/ra-tls-server/Makefile

```diff
@@ -12,22 +12,17 @@ SGX_SDK ?= /opt/intel/sgxsdk
 SGX_RA_TLS := $(TOPDIR)/sgx-ra-tls
 
 ifdef ECDSA
-  SGX_DCAP_URI := https://github.com/intel/SGXDataCenterAttestationPrimitives
-  SGX_DCAP_COMMIT := bfab1376480f760757738092399d0d99b22f4dfd
   SGX_DCAP ?= SGXDataCenterAttestationPrimitives
-  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include
+  SGX_DCAP_INC := -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/common/inc -I$(SGX_DCAP)/QuoteGeneration/pce_wrapper/inc -I$(SGX_DCAP)/QuoteVerification/Src/AttestationLibrary/include -I$(SGX_DCAP)/QuoteGeneration/quote_wrapper/ql/inc -I../build/include/ -I/usr/x86_64-linux-gnu/include/ -I$(SGX_DCAP)/QuoteVerification/QVL/Src/AttestationLibrary/include/ -I/usr/include/x86_64-linux-gnu/
 endif
 
-#EPID_SDK := $(SGX_SDK)/external/epid-sdk
-
-CFLAGS += -std=gnu99 -I$(SGX_RA_TLS) -I$(SGX_SDK)/include -I$(INCDIR) $(SGX_DCAP_INC) -fPIC
-#CFLAGSERRORS := -Wall -Wextra -Wwrite-strings -Wlogical-op -Wshadow -Werror
+CFLAGS += -std=gnu99 -I$(SGX_RA_TLS) -I$(SGX_SDK)/include -I$(INCDIR) $(SGX_DCAP_INC)
```

Comment:
Why removing -fPIC?

Reply:
had pie in compiling options. It's a repeated parameters

Reply:
But -fpic and -fpie both take effect on CFLAGS. So you should use -fPIE in CFLAGS.

Reply:
GO_BUILD := CGO_ENABLED=1 $(GO) build $(MOD_VENDOR) **-buildmode=pie** $(GCFLAGS) $(EXTRA_FLAGS) \

Reply:
```C
int v = 1;

int test()
{
	printf("%d\n", v);
}
```

You can see the differences w/ and w/o -fPIC/-fPIE in CFLAGS:

```
0000000000000000 <test>:
   0:	55                   	push   %rbp
   1:	48 89 e5             	mov    %rsp,%rbp
   4:	8b 05 00 00 00 00    	mov    0x0(%rip),%eax        # a <test+0xa>
   a:	89 c6                	mov    %eax,%esi
   c:	bf 00 00 00 00       	mov    $0x0,%edi
  11:	b8 00 00 00 00       	mov    $0x0,%eax
  16:	e8 00 00 00 00       	callq  1b <test+0x1b>
  1b:	90                   	nop
  1c:	5d                   	pop    %rbp
  1d:	c3                   	retq   
```

```
0000000000000000 <test>:
   0:	55                   	push   %rbp
   1:	48 89 e5             	mov    %rsp,%rbp
   4:	8b 05 00 00 00 00    	mov    0x0(%rip),%eax        # a <test+0xa>
   a:	89 c6                	mov    %eax,%esi
   c:	48 8d 3d 00 00 00 00 	lea    0x0(%rip),%rdi        # 13 <test+0x13>
  13:	b8 00 00 00 00       	mov    $0x0,%eax
  18:	e8 00 00 00 00       	callq  1d <test+0x1d>
  1d:	90                   	nop
  1e:	5d                   	pop    %rbp
  1f:	c3                   	retq
```